account/test/e2e/createUserAccount.test.js

```diff
@@ -0,0 +1,34 @@
+import { app } from "../../src/app.js";
+import request from "supertest";
+import { client, getUsersCollection } from "../../src/repositories/accountRepository.js";
+
+
+
+describe("POST em accounts", () => {
+  afterEach( async() => {
+    await client.connect();
```

Comment:
do you think this is the right place to create the connection?

Reply:
as a good practice, after each test always connect, clean and close the bank, so as not to interfere with other tests.